src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -111,8 +114,9 @@ public static void WarmUpCacheInCurrentProcess(Project project)
 
             // First find symbols of all applicable extension methods.
             // Workspace's syntax/symbol index is used to avoid iterating every method symbols in the solution.
-            var symbolComputer = await SymbolComputer.CreateAsync(
-                document, position, receiverTypeSymbol, namespaceInScope, cancellationToken).ConfigureAwait(false);
+            semanticModel ??= await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            var symbolComputer = new SymbolComputer(
```

Comment:
SymbolComputer.CreateAsync can be deleted now

Reply:
➡️ Removed when I fixed the merge conflict